CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 # Changelog
 
-## 1.0-21 (In Development)
+## 1.0-21
```

Comment:
Is that correct version? Looks like Xcode don't have build number `21`?
Same question for [Release/0.4.0 to master](https://github.com/mapbox/vision-ios-examples/pull/78/)

Reply:
Yep, version is set before pushing to testflight by incrementing last one. There's a plan to add fastlane automation for that.